cli/tools/jupyter/server.rs

```diff
@@ -480,57 +480,27 @@ async fn get_jupyter_display(
   session: &mut repl::ReplSession,
   evaluate_result: &cdp::RemoteObject,
 ) -> Result<Option<HashMap<String, serde_json::Value>>, AnyError> {
-  let mut data = HashMap::default();
   let response = session
     .call_function_on_args(
       r#"function (object) {{
-        return object[Symbol.for("Jupyter.display")]();
-      }}"#
+      return JSON.stringify(object[Symbol.for("Jupyter.display")]());
+    }}"#
         .to_string(),
       &[evaluate_result.clone()],
     )
     .await?;
 
-  if response.exception_details.is_some() {
-    return Ok(None);
-  }
-
-  let object_id = response.result.object_id.unwrap();
-
-  let get_properties_response_result = session
-    .post_message_with_event_loop(
-      "Runtime.getProperties",
-      Some(cdp::GetPropertiesArgs {
-        object_id,
-        own_properties: Some(true),
-        accessor_properties_only: None,
-        generate_preview: None,
-        non_indexed_properties_only: Some(true),
-      }),
-    )
-    .await;
+  let json_str = response.result.clone().value.unwrap_or_default();
```

Comment:
Let's change this to an if let/else block so we can ensure this error at least gets logged:

```rust
if let Some(json_str) = response.result.clone().value else {
  eprintln!("Failed to read JSON response from Jupyter.display");
  return Ok(None)l;
};
```

Reply:
I log it in the else block below as the raw item. I'll move that down into there.